camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlRecordReader.java

```
@@ -252,10 +251,14 @@ public boolean nextKeyValue() throws IOException, InterruptedException {
                     CamusWrapper wrapper;
                     try {
                         wrapper = getWrappedRecord(key.getTopic(), bytes);
-                    } catch (Exception e) {
+                    } catch (IOException e) {
+                        exceptionCount++;
+                        long errorSchemaThresold = getSchemaErrorsThresold(context);
+                        if (errorSchemaThresold > 0 && errorSchemaThresold <= exceptionCount) {
```

Default condition should be errorSchemaThreshold == 0.  Thus, the job will fail upon schema error.
If we want to relax that condition, we can set it to a positive value, e.g.  errorSchemaThreshold == 5 means that up to 5 errors are swallowed; 6th error fails the job.